userdocs/diagnostics/deprecated-declaration.md

```diff
@@ -1,7 +1,6 @@
-# Deprecated Declaration Warnings (`DeprecatedDeclaration`)
+# Deprecated declaration warnings (`DeprecatedDeclaration`)
```

Comment:
DocC doesn't handle code voice in titles well. Should we move the group names elsewhere?

Reply:
Hmmm, what if I just remove the backticks? Maybe the parentheses are clear enough?

Reply:
Another alternative - make the group name the summary text?